### ISSUE TITLE
Consistent prediction logits

### ISSUE THREAD

**Issue by daniel-ranti** [kexinhuang12345/clinicalBERT#7]
We are attempting to use custom pretrained parameters for the MIMIC III 30-day readmission task, and getting some odd behavior, namely that the model predicts consistent logits across all eval examples (see screenshot below). We running `run_readmission.py` with the start command below. We attempted the same task with the `bert-base-uncased` parameters, and got slightly different logits (by 0.01), but still consistent across all eval examples. The only thing we changed about the `run_readmission.py` script was `from sklearn.utils.fixes import signature` to `from inspect import signature` based on deprecation issues. 

Has anyone run into this issue before?

![image](https://user-images.githubusercontent.com/20704622/75163988-1dc6e280-56ee-11ea-9e28-dbe1b412ad53.png)

```
python ./clinicalBERT/run_readmission.py \
          --task_name readmission \
          --do_train \
          --do_eval \
          --data_dir ./mimic_iii_readmissions_data/discharge \
          --readmission_mode discharge_summary \
          --bert_model <path to our pretrained model> \
          --max_seq_length 512 \
          --train_batch_size 4 \
          --learning_rate 2e-5 \
          --num_train_epochs 1 \
          --output_dir <output dir>```

**Comment by kexinhuang12345** [comment]
Hi, sorry for the late reply! Could you share more details? Are you using the same model checkpoints and running the same data preprocessing scheme? Or are you using your own pre-trained checkpoints?  Is it after finetuned? If you can, could you examine some sample inputs?  It is likely that the classifier on top of BERT is not learning anything.

**Comment by kexinhuang12345** [comment]
Closed for now, reopen if you still have questions.

**Comment by elehman16** [comment]
To all future individuals reading this issue: this is a bug in the code. The model weights are not being loaded properly, which thus causes the model to predict all uniform outputs.

**Comment by manels7** [comment]
@elehman16 did you manage to fix this issue? If so, could you please mention what you changed? Thanks
